charts/internal/alicloud-infra/values.yaml

```diff
@@ -1,26 +1,27 @@
 alicloud:
   region: cn-beijing
 
-create:
-  vpc: true
-
 clusterName: test-namespace
 
 sshPublicKey: sshkey-12345
 
 vpc:
+  create: true
   id: alicloud_vpc.vpc.id
   cidr: 10.10.10.10/6
+
+natGateway:
   natGatewayID: alicloud_nat_gateway.nat_gateway.id
-  snatTableID: alicloud_nat_gateway.nat_gateway.snat_table_ids
+  sNatTableIDs: alicloud_nat_gateway.nat_gateway.snat_table_ids
+
+eip:
   internetChargeType: PayByTraffic
 
 zones:
 - name: cn-beijing-a
   cidr:
     workers: 10.250.0.0/19
-#  natGateway:
-#    eipAllocationID: eip-ufxsdg122elmszcg
+#  eipAllocationID: eip-ufxsdg122elmszcg
```

Comment:
```suggestion
#   eipAllocationID: eip-ufxsdg122elmszcg
```

Reply:
@rfranzke thanks for your detailed reviews. Other comments were addressed, but I didn't get your point here. Why need one more indent? The `eipAllocationID` should be aligned with `cidr`.

Reply:
Sure, ignore it, thanks!